src/types/FileT.ts

```diff
@@ -0,0 +1,26 @@
+class FileT implements SBVRType<Buffer, any> {
+	types = {
+		  postgres: 'BYTEA'
+		, mysql: 'BLOB'
+		, websql: 'BLOB'
+		, odata: {
+			name: 'Edm.String' // TODO: What should this really be?
+		}
+	}
+	validate = (value:any, required:boolean, callback:Callback<Buffer>) => {
```

Comment:
Do you need to manually specify these types? Can't they be picked up by the `implements SBVRType<Buffer, any>` association saying what the type of this method needs to be?